assets/pages/oneoff-contributions/reducers/reducers.js

```diff
@@ -39,6 +40,11 @@ export type CombinedState = {
   csrf: CsrfState,
 };
 
+export type PageState = {
+  common: CommonState,
+  page: CombinedState,
+}
+
 // ----- Reducers ----- //
 
 function createOneOffContribReducer(amount: number, currency: Currency, country: IsoCountry) {
```

Comment:
I don't think `currency` and `country` need to be handled by this reducer any more, as they exist in the `common` state.

Reply:
Actually, currency doesn't, but it's a translation from country using `forCountry` in `currency.js`.